scripts/init

```diff
@@ -154,27 +146,51 @@ craft_wallarm_status_snippet() {
 }
 
 insert_wallarm_status_snippet() {
-  line_for_insert=$(grep -n "location / {" /etc/nginx/sites-enabled/default | cut -f 1 -d:)
+  line_for_insert=$(grep -n "location / {" /etc/nginx/conf.d/default.conf | cut -f 1 -d:)
   line_for_insert=$((line_for_insert - 1))
-  sed -i "${line_for_insert}r /tmp/wallarm-status.conf" /etc/nginx/sites-enabled/default
+  sed -i "${line_for_insert}r /tmp/wallarm-status.conf" /etc/nginx/conf.d/default.conf
 }
 
 configure_services() {
+  mkdir -p /etc/supervisor
+
   if [ x"$POSTANALYTIC_ONLY" = x"true" ] \
     || [ x"$POSTANALYTIC_ONLY" = x"True" ] \
     || [ x"$POSTANALYTIC_ONLY" = x"yes" ]
   then
-    rm /etc/cron.d/wallarm-node-nginx
-    rm /etc/collectd/wallarm-collectd.conf.d/nginx-wallarm.conf
-    mv /etc/supervisor/supervisord.post-analytics.conf.example /etc/supervisor/supervisord.conf
-    sed  -e 's/# post-analytic://' -i /etc/cron.d/wallarm-node-tarantool
+    #rm /etc/cron.d/wallarm-node-nginx
+    rm /opt/wallarm/etc/collectd/wallarm-collectd.conf.d/nginx-wallarm.conf
+    mv /opt/wallarm/etc/supervisord.conf.postanalytics /opt/wallarm/etc/supervisord.conf
+    #sed  -e 's/# post-analytic://' -i /etc/cron.d/wallarm-node-tarantool
   fi
 
   if [ -n "$POSTANALYTIC_ADDRESS" ]; then
-    rm /etc/cron.d/wallarm-node-tarantool
-    rm /etc/collectd/wallarm-collectd.conf.d/wallarm-tarantool.conf
-    mv /etc/supervisor/supervisord.filtering.conf.example /etc/supervisor/supervisord.conf
+    #rm /etc/cron.d/wallarm-node-tarantool
+    rm /opt/wallarm/etc/collectd/wallarm-collectd.conf.d/wallarm-tarantool.conf
+    mv /opt/wallarm/etc/supervisord.conf.filtering /opt/wallarm/etc/supervisord.conf
+
+    # add socat block
+    cat <<EOF >> /opt/wallarm/etc/supervisord.conf
+
+[program:socat]
+command=socat TCP4-LISTEN:3313,fork,reuseaddr TCP4:%(ENV_POSTANALYTIC_ADDRESS)s:3313
+autorestart=true
+startretries=4294967295
+redirect_stderr=true
+stdout_logfile=/opt/wallarm/var/log/wallarm/socat.log
+EOF
+
   fi
+
+  cat <<EOF >> /opt/wallarm/etc/supervisord.conf
+
+[program:registernode_loop]
```

Comment:
AFAIK we decided to get rid of `registernode_loop`, right?

Reply:
Dropped it now